recipes/pantax/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    cp -rf $SRC_DIR $PREFIX
+else
+    cd ${SRC_DIR}/scripts
+    chmod +x pantax
+    chmod +x data_preprocessing
+    cp ${SRC_DIR}/scripts/pantax ${SRC_DIR}/scripts/data_preprocessing ${PREFIX}/bin
+    cp ${SRC_DIR}/scripts/*py ${PREFIX}/bin
+
+    mkdir -p ${PREFIX}/bin/tools
+    cd ${SRC_DIR}/tools/fastix
+    cargo install fastix --root ./
+    cp ${SRC_DIR}/tools/fastix/bin/fastix ${PREFIX}/bin/tools
```

Comment:
Why do you rename `fastix` to `tools` ?
I think this way there is a bigger chance for name collision

Reply:
I create a directory with the command `mkdir -p ${PREFIX}/bin/tools`. `${PREFIX}/bin/tools` is a directory. I just copy the binary to the directory. But now I install the binary to `$PREFIX/bin`, this is not a problem.